dsgrid/cli/config.py

```diff
@@ -62,7 +62,7 @@ def config():
 def create(
     timings,
     database_name,
-    database_url,
```

Comment:
Shouldn't this function also add user name and password options?

Reply:
Yeah, good point. Fixed.